src/rpc/handlers/Tx.cpp

```diff
@@ -36,6 +36,15 @@ doTx(Context const& context)
     if (!range)
         return Status{Error::rpcNOT_READY};
 
+    if (request.contains(JS(min_ledger)) && request.contains(JS(max_ledger)))
+    {
+        const auto& minLedger = request.at(JS(min_ledger));
+        const auto& maxLedger = request.at(JS(max_ledger));
+
+        if ((maxLedger.as_int64() - minLedger.as_int64()) > 1000)
```

Comment:
Consider adding a check for `max > min` as well now that we have this check.

Reply:
Btw. we are checking these fields but are they even used anywhere? I could not find any mentions of `min_ledger` in the entire codebase.

Reply:
@godexsoft min_ledger and max_ledger are parseable fields in the request that Mounika outlined here 
![image](https://user-images.githubusercontent.com/47958858/202021395-e2891e62-9660-4c8f-b5ef-f0a0a4ea3e36.png)